### PR TITLE
Expand Buildkite section on error

### PR DESCRIPTION
**Pull request by pecigonzalo**
Expand sub-sections of parallel runs to make it easier to spot the errors.

### CODE REVIEW

File: dev/foreach-ts-project.sh [hunk 1]
```diff
@@ -33,6 +33,9 @@ run_command() {
   (
     set -x
     cd "$dir" && eval "${MAYBE_TIME_PREFIX} ${ARGS}"
+    if [[ $? -ne 0 ]]; then
+      echo "^^^ +++"
+    fi
```
Comment (felixfbecker) [inline dev/foreach-ts-project.sh:38]: Could this be made so it's not logged by `set -x` that is active in the `()`? Would `$?` be accessible outside the `()`? (I'm not a bash pro)

**Review by felixfbecker — APPROVED** [review]
Perfect! ❤️

File: dev/foreach-ts-project.sh (hunk 1 above)
Comment (pecigonzalo) [reply dev/foreach-ts-project.sh:38]: I can `set +x` before print.

Comment (pecigonzalo) [reply dev/foreach-ts-project.sh:38]: Yes, it will catch it outside of `()` in this case.

**Pull request merged**